devito/ir/iet/nodes.py

```diff
@@ -465,7 +465,8 @@ def __init__(self, name, body, retval, parameters=None, prefix=('static', 'inlin
         self.parameters = as_tuple(args)
 
     def __repr__(self):
-        parameters = ",".join([c.dtype_to_ctype(i.dtype) for i in self.parameters])
+        parameters = ",".join(['void*' if i.is_PtrArgument else c.dtype_to_ctype(i.dtype)
```

Comment:
Presumably, `dtype_to_ctype` is missing pointer types completely? Should we wrap it then in a separate utility to add pointer types, instead of relying on individual special-casing when it is used?

Reply:
Can we consider this for a future PR?

Reply:
yes